qa/suites/rgw/multifs/omap_limit/10.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        osd_max_omap_entries_per_request: 10
```

Comment:
looks like the problem is in `cls_user_list_buckets()`, which assumes that `cls_cxx_map_get_vals()` will give it all of the entries it asked for. changing the calculation of `ret.truncated` so that it depends on `max_entries` rather than `keys.end()` fixes the test:
```patch
diff --git a/src/cls/user/cls_user.cc b/src/cls/user/cls_user.cc
index c616d1a..66c1300 100644
--- a/src/cls/user/cls_user.cc
+++ b/src/cls/user/cls_user.cc
@@ -328,7 +328,7 @@ static int cls_user_list_buckets(cls_method_context_t hctx, bufferlist *in, buff
     }
   }
 
-  if (iter == keys.end())
+  if (i == max_entries)
     done = true;
   else
     ret.marker = marker;
```
i worry that there are similar issues in other uses of `cls_cxx_map_get_vals()` though